variables.tf

```diff
@@ -212,3 +219,9 @@ variable "prometheus_enabled" {
   default     = false
   type        = bool
 }
+
+//[ { root_url = "alluxio://alluxio1", s3_buckets = "bucket1,bucket2" }, { root_url = "alluxio://alluxio2", s3_buckets = "bucket3,bucket4" } ]
+variable "alluxio_endpoints" {
```

Comment:
This variable needs to also be documented in the `Variables` section of `README.md`.

Reply:
done